ofx.go

```diff
@@ -46,11 +47,16 @@ const (
 )
 
 type Amount struct {
-	value big.Rat
```

Comment:
Although the underlying type is a `big.Rat`, its convenient to just use `value`.